pkgs/development/libraries/appstream/default.nix

```diff
@@ -49,6 +50,12 @@ stdenv.mkDerivation rec {
     ./installed-tests-path.patch
   ];
 
+  strictDeps = true;
+
+  depsBuildBuild = [
+    pkg-config
+  ];
```

Comment:
Is this really needed here? (And since `pkg-config` doesn't prefix/suffix executables by target, wouldn't this shadow the `pkg-config` in `PATH` from `nativeBuildInputs`, since it's in both?)

I can build `pkgsCross.riscv64.appstream` just fine with `depsBuildBuild` removed on x86_64-linux and I'm not sure it should theoretically be here either given pkg-config doesn't have a concept of targets like compilers do

But I may also just be misunderstanding since my cross knowledge is admittedly quite limited still. Thoughts?

Reply:
It is not *really* needed, as `meson` fallbacks find tools in `PATH` if `pkg-config` is not available (you can see that from build log). And `pkg-config` is indeed prefixed just as compilers do, and it does has a concept of targets.

Reply:
> And `pkg-config` is indeed prefixed just as compilers do, and it does has a concept of targets.

Ah, so it does! Not sure how I misread that the first time

Ignore me, I'm dumb :)